src/server/api/interaction-step.js

```diff
@@ -14,8 +19,19 @@ export const resolvers = {
   InteractionStep: {
     ...mapFieldsToModel([
       'id',
-      'script'
+      'script',
+      'answerOption',
+      'answerActions',
+      'parentInteractionId',
+      'question',
+      'isDeleted'
     ], InteractionStep),
+    questionText: async(interactionStep) => {
```

Comment:
When I run this, using dynamic assignment, I get the following error on the texter contact screen:

Client:
```
minilog.js:17 client  
ApolloError {graphQLErrors: Array(1), networkError: TypeError: Cannot read property 'interactionSteps' of undefined at new AssignmentTexterContact (…, message: "GraphQL error: Cannot read property '0' of undefin…not read property 'interactionSteps' of undefined"}
graphQLErrors
:
[{…}]
message
:
"GraphQL error: Cannot read property '0' of undefined↵Network error: Cannot read property 'interactionSteps' of undefined"
networkError
:
TypeError: Cannot read property 'interactionSteps' of undefined at new AssignmentTexterContact (http://localhost:3000/assets/bundle.js:211469:83) at ReactCompositeComponentWrapper._constructComponentWithoutOwner (http://localhost:3000/assets/bundle.js:23568:28) at ReactCompositeComponentWrapper._constructComponent (http://localhost:3000/assets/bundle.js:23550:22) at ReactCompositeComponentWrapper.mountComponent (http://localhost:3000/assets/bundle.js:23469:22) at Object.mountComponent (http://localhost:3000/assets/bundle.js:21841:36) at ReactCompositeComponentWrapper.performInitialMount (http://localhost:3000/assets/bundle.js:23658:35) at ReactCompositeComponentWrapper.mountComponent (http://localhost:3000/assets/bundle.js:23532:22) at Object.mountComponent (http://localhost:3000/assets/bundle.js:21841:36) at ReactCompositeComponentWrapper.performInitialMount (http://localhost:3000/assets/bundle.js:23658:35) at ReactCompositeComponentWrapper.mountComponent (http://localhost:3000/assets/bundle.js:23532:22)
__proto__
:
Error
```

Server:
```
3:45:06 PM server.1               |  TypeError: Cannot read property '0' of undefined
3:45:06 PM server.1               |      at getTopMostParent (src/lib/interaction-step-helpers.js:90:10)
3:45:06 PM server.1               |      at _callee7$ (src/server/api/campaign-contact.js:168:14)
3:45:06 PM server.1               |      at tryCatch (/Users/shaka/moveon/Spoke/node_modules/regenerator-runtime/runtime.js:62:40)
3:45:06 PM server.1               |      at GeneratorFunctionPrototype.invoke [as _invoke] (/Users/shaka/moveon/Spoke/node_modules/regenerator-runtime/run
3:45:06 PM server.1               |  >  time.js:336:22)
```

When I add some console.logs, it looks like `allInteractionSteps` here isn't updating in the getAvailableInteractionSteps() function in the `AssignmentTexterContact` component. It remains an empty array which then get's passed to  `getTopMostParent` helper function.

Reply:
I believe this will be fixed with 637c8e5